### ISSUE TITLE
Can't build docker image in a docker:dind container

### ISSUE THREAD

**Issue by roussidis** [nestybox/sysbox#476]
Trying to build docker images inside a docker:dind

The idea was inspired from [this](https://blog.nestybox.com/2020/10/21/gitlab-dind.html) blog post. 

Host docker version 20.10.12, sysbox version 0.4.1 and shiftfs enabled.
To reproduce this: 

1. Create a docker:dind container.
```
docker run -d --runtime=sysbox-runc --name docker_builder docker:dind
```

2. Enter the container and create a dummy Dockerfile and try to build out of that Dockerfile.
```
docker exec -it docker_builder sh
vim Dockerfile
docker build -t test_build .
```

The output error is `error checking context: 'file ('/proc/351/fs/7') not found or excluded by .dockerignore`

The above error was reproduced in VMs with ubuntu focal and debian 11 OSes.
Sysbox was installed in both of them with the official packages that exist.

**Comment by rodnymolina** [comment]
@roussidis, I'm unable to reproduce this one following the steps above.

Can you please share the `Dockerfile` that you utilize to reproduce the docker-build issue?

**Comment by roussidis** [comment]
Hi @rodnymolina thank you for your response. 

While I was testing it once again I found the problem. I was trying to build the image on the root directory so the location of the Dockerfile was at /Dockerfile and at this location I can not build the image. When I moved it to the home directory of the root user inside the container I was able to build it.

This looks like the default behavior from docker:dind image and not sysbox related as it was reproduced without the sysbox-runc runtime. Probably it can be closed. 

Sorry for wasting your time
